gprofiler/main.py

```diff
@@ -602,6 +605,30 @@ def parse_cmd_args() -> configargparse.Namespace:
         " beginning of a session.",
     )
 
+    parser.add_argument(
+        "--limit-memory",
+        default=(1 << 30),  # 1Gi, same as in the k8s DaemonSet
+        dest="memory_limit",
+        type=int,
+        help="Limit on the memory used by gProfiler."
+    )
+
+    parser.add_argument(
```

Comment:
How do I specific no CPU limit? i.e `--limit-cpu none` should be a possible parameter, not necessarily this way, but you should be able to limit ONLY the memory or CPU.